### DISCUSSION TITLE
Feb 9 Lecture

### DISCUSSION THREAD

**Discussion by noreengao0924** [chulminy/CIVE497-CIVE700#147] [General]
For the Feb 9 lecture, are we skipping class 07 and reviewing class 08 directly? what about class 07?  
Also, are assignment solutions getting posted?

**Comment by chulminy** [comment]
Yes. The lecture video for class07 is sufficient. I will review Class 08.

Solutions will not be posted. I will just post a model report. 

Chul Min

**Reply by noreengao0924** [reply]
Thank you.
